test/Microsoft.AspNetCore.Blazor.Build.Test/RazorIntegrationTestBase.cs

```diff
@@ -60,11 +61,11 @@ public RazorIntegrationTestBase()
 
             Configuration = BlazorExtensionInitializer.DefaultConfiguration;
             FileSystem = new VirtualRazorProjectFileSystem();
+            PathSeparator = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "\\" : "/";
```

Comment:
Incredibly minor point, but for my own education, how is this different from `Path.PathSeparator`?

Reply:
We need the path logic to work on the same way on Windows and Linux for the baseline based tests because the filenames are backed into the tests themselves. Methods like `Path.GetFilename` and `Path.Combine` won't work with windows paths on linux.

Reply:
I get that, I just wondered whether there was any reason to evaluate `RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "\\" : "/"` instead of using `Path.DirectorySeparatorChar` which AFAIK would return the same platform-sensitive character. Not at all important though!

Reply:
Ah, I missed what you were getting at. Sure that's right. I was doing this on my mac without intellisense.